lib/hotwire_combobox/helper.rb

```diff
@@ -166,8 +166,8 @@ def hw_combobox_page_stream_action
 
       def hw_uri_with_params(url_or_path, **params)
         URI.parse(url_or_path).tap do |url_or_path|
-          query = URI.decode_www_form(url_or_path.query || "").to_h.merge(params)
-          url_or_path.query = URI.encode_www_form query
+          query = URI.decode_www_form(url_or_path.query || "").to_h.with_indifferent_access
+          url_or_path.query = URI.encode_www_form query.merge(params)
```

Comment:
Is this so that params are properly overridden by params provided as symbols? If so, I think let's just treat all keys as strings here. So we're not guessing which keys exist at any time:

```suggestion
          query = URI.decode_www_form(url_or_path.query || "").to_h.merge(params.stringify_keys)
          url_or_path.query = URI.encode_www_form query
```

Also, let's do that in the `query = ` assignment (as above) so that it contains the final query with no need for modification on the next line.

Quick test to make sure that works:

```rb
URI.decode_www_form(URI.parse("google.com?foo=fooish&bar=barish").query).to_h.merge({ bar: "baz" }.stringify_keys)
#=> {"foo"=>"fooish", "bar"=>"baz"}
```

Reply:
Yes.
Because params is hash with symbols


<img width="977" alt="Screenshot 2024-03-31 at 17 04 34" src="https://github.com/josefarias/hotwire_combobox/assets/7427365/bb1d2b1e-f322-41c8-9a9f-2194e1746575">



All of these will work:
<img width="1228" alt="Screenshot 2024-03-31 at 17 05 49" src="https://github.com/josefarias/hotwire_combobox/assets/7427365/c506720b-5220-4ce9-96bb-05d9c3cb5b74">

We can do `stringify_keys` if you prefer that one